src/SimpleThings/EntityAudit/AuditConfiguration.php

```diff
@@ -35,6 +37,23 @@ class AuditConfiguration
     private $currentUsername = '';
     private $revisionIdFieldType = 'integer';
 
+    /**
+     * @param ClassMetadataInfo $metadata
+     *
+     * @return string
+     */
+    public function getTableName(ClassMetadataInfo $metadata)
+    {
+        $tableName = $metadata->getTableName();
+
+        //## Fix for doctrine/orm >= 2.5
+        if (method_exists($metadata, 'getSchemaName') && $metadata->getSchemaName()) {
+            $tableName = $metadata->getSchemaName() . '.' . $tableName;
+        }
+
+        return $this->getTablePrefix() . $tableName . $this->getTableSuffix();
```

Comment:
Doesn't this concatination need to be done before prefixing the schema name? Else you might end up having `$tablePrefix.$schemaName.$tableName.$tableSuffix`